go.mod

```diff
@@ -182,16 +185,10 @@ require (
 replace (
 	github.com/facebookincubator/nvdtools => github.com/stackrox/nvdtools v0.0.0-20231111002313-57e262e4797e
 
-	// Our fork has following features:
-	// - generate MessageClone() and Clone() functions
-	// - error on nil unmarshal instead of panic
-	// - trailing comments propagation to generated code
-	github.com/gogo/protobuf => github.com/stackrox/protobuf v1.2.2-0.20240207122816-e936d453291c
-
 	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.1.0
 
 	github.com/operator-framework/helm-operator-plugins => github.com/stackrox/helm-operator v0.0.12-0.20230825152000-1361e2f7db46
-	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20240402171531-15fa6d254174
+	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20240510180132-d8b1b7d8d248
```

Comment:
I think it would be good to update it to latest master version in a separated PR to minimize diff.
Also update to master could be merged right now.